subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -4516,7 +4531,7 @@ static inline int length_req_rsp_recv(struct ll_conn *conn, memq_link_t *link,
 #if defined(CONFIG_BT_CTLR_PHY_CODED)
 			eff_rx_time = MIN(max_tx_time,
 					  PKT_US(LL_LENGTH_OCTETS_RX_MAX,
-						 BIT(2)));
+						 conn->lll.phy_rx));
```

Comment:
delete lines 4538-4540. Move line 4531 after 4534. This is how its in legacy controller (keep your change in line 4534).

Reply:
I will do so, but first I'll push an update which should work OK in Denmark; hopefully they can test while I rewrite the code as per your suggestion.